src/app/startup_portfolio/page.js

```diff
@@ -0,0 +1,81 @@
+import incubatees from "./_data/incubatees";
+import graduates from "./_data/graduates";
+import spark from "@/assets/logo-spark.svg";
+import fb from "@/assets/facebook-white-icon.svg";
+import { EnvelopeIcon } from "@heroicons/react/24/solid"
+import Image from 'next/image';
+
+
+
+export default function StartupPortfolio() {
+  return (
+    <div className="flex flex-col items-center justify-center gap-12 sm:gap-6 w-full max-w-screen py-10 sm:py-5">
+        {/* TITLE */}
+        <section className="mt-[6rem] w-full max-w-[60rem] px-8 sm:px-4">
+            <h1 className="text-center">Startup Portfolio</h1>
+            <p className="mt-6 text-justify">
+                Since its 2015 inauguration, BINHI has fostered the development of more than 20 startups within Iloilo. Presented herein are each one of them as current incubatees and graduates.
+            </p>
+        </section>
+
+        {/* CURRENT INCUBATEES */}
+        {/* FIXING IT INTO A GRID IN THE FUTURE */}
+        <section className="flex flex-col gap-12 mt-[5rem] sm:pt-6 px-5 sm:p-2 w-full max-w-[75rem]">
+            <h2 className="text-center">Current Incubatees</h2>
+            <div className="flex flex-wrap items-center justify-center gap-y-10 gap-x-20">
+                { 
+                    incubatees.map((incubatee, index) => (
```

Comment:
With the suggestion I gave sa `incubatees.js` earlier, you could do it like this: 
```suggestion
                    Object.entries(incubatees).map(([incubatee, img], index) => (
```